### PR TITLE
Static files fall back docs clarification 2609

### PR DESCRIPTION
**Pull request by Santoshziffity**
## Purpose of this pull request

This pull request (PR) for Remove the wrong path of the static file 

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-inherit.html

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]

**Comment by dobooth** [comment]
Hi @Santoshziffity. Thanks for contributing.  Can you please fix this merge conflict?

**Comment by Santoshziffity** [comment]
> Hi @Santoshziffity. Thanks for contributing. Can you please fix this merge conflict?

**Comment by dobooth** [comment]
@BarnyShergold Here's another OrangeCo instance. What are you using instead?

File: src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md [hunk 1]
```diff
@@ -71,8 +71,7 @@ If module context is not defined for a file:
 
 If module context is defined for a file:
 
-1. Current theme and current locale module static files: `<theme_dir>/web/i18n/<locale>/<Namespace>_<Module>`
-1. Current theme module static files `<theme_dir>/<Namespace>_<Module>/web/`. Example: `app/design/frontend/SampleCompany/orange/Magento_Catalog/web/`
+1. Current theme module static files `<theme_dir>/<Namespace>_<Module>/web/`.Example: `app/design/frontend/OrangeCorp/orange/Magento_Catalog/web/`
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md:74]:
```suggestion
1. Current theme module static files `<theme_dir>/<Namespace>_<Module>/web/`.Example: `app/design/frontend/ExampleCorp/orange/Magento_Catalog/web/`
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md (hunk 1 above)
Comment (BarnyShergold) [reply src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md:74]: We just need to clean this one file up to remove OrangeCorp

**Review by BarnyShergold — APPROVED** [review]
Thanks for being so quick!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**